src/scripts.js

```diff
@@ -97,6 +115,145 @@ function getUserById(id, listRepo) {
   return listRepo.getDataFromID(id);
 }
 
+function windowOnClick(event) {
+  if (event.target.classList.contains('new-hydration-button')) {
+    submitNewHydration(userNowId);
+  }
+  if (event.target.classList.contains('new-activity-button')) {
+    submitNewActivity(userNowId);
+  }
+  if (event.target.classList.contains('new-sleep-button')) {
+    submitNewSleep(userNowId);
+  }
+}
+
+function getTodaysDate() {
+  let today = new Date();
+  let dd = String(today.getDate()).padStart(2, '0');
+  let mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+  let yyyy = today.getFullYear();
+  return today = yyyy + '/' + mm + '/' + dd;
+}
+
+function submitNewHydration(userNowId) {
+  // let todaysDate = getTodaysDate();
+  let todaysDate = '2020/9/31';
+  /*
+  This is to remind me that if there are multiple entries on the same day, see not on line 163!!! 
+  */
+  let postedHydration = requests.postHydrationData(userNowId, todaysDate, +newHydrationInput.value);
+  Promise.all([postedHydration])
+    .then(() => {
+      updatePageHydration(userNowId, todaysDate);
+    });
+  newHydrationButton.disabled = true;
+}
+
+function updatePageHydration(userNowId, todaysDate) {
+  let newRecievedHydration = requests.fetchHydrationData();
+  Promise.all([newRecievedHydration])
+    .then((value) => {
+      hydrationData = value[0];
+      const newHydration = new Hydration(hydrationData);
+      let hydrationElementsToClear = [hydrationToday]; //, hydrationAverage, hydrationThisWeek, hydrationEarlierWeek]
+      clearHtml(hydrationElementsToClear);
+      // clearHtml();
+      // startApp(userNowId);
+      addDailyOuncesInfo(userNowId, newHydration, todaysDate);
+      /*
+      NOTE: These only work if there is are NOT multiple data for the same date for any given user 
+      */
+    });
+}
+
+function submitNewActivity(userNowId) {
+  // let todaysDate = getTodaysDate();
+  let todaysDate = '2020/9/31';
+  let newStepsActivityData = newStepsInput.value;
+  let newMinutesActivityData = newStairsInput.value;
+  let newStairsActivityData = newActiveMinutesInput.value;
+  if (!newStepsActivityData) {
+    return alert('Please enter a number of steps');
+  } else if (newStepsActivityData > 10000) {
```

Comment:
Thanks for adding all these conditionals Amanda! I think I made the suggestion of limiting to 10,000 steps, but maybe we should amend that. I know I have done some activities that put me WAY over that number (I think my max ever was somewhere around 32,000). This definitely isn't a big deal though! The code logic is sound, and that is what matters! :)